action.yml

```
@@ -8,8 +8,8 @@ inputs:
     default: ${{ github.token }}
     required: false
   github_token_actor:
-    description: 'The name of the GitHub app that owns the GitHub API Access Token (see github_token). Used to identify pull request comments created by this action during earlier runs. Has to be set when `github_token` is set to a GitHub app installation token (other than GitHub actions). Otherwise, existing comments will not be updated, but new comments created. Note: this does not change the bot name of the pull request comments. Defaults to "github-actions".'
-    default: 'github-actions'
+    description: '[deprecated] This is no longer considered at all without replacement'
+    deprecationMessage: 'This is no longer considered at all without replacement'
```

Nice that there is `deprecationMessage` now:
```suggestion
    deprecationMessage: 'This is not needed any more as this is detected automatically.'
```

---

Yeah, since October 2023 :-)
Quite handy, especially when you create bindings like with https://github.com/typesafegithub/github-workflows-kt/ and there get actual deprecation warnings at compile time already. :-)